renderer/main.js

```diff
@@ -1213,11 +1213,7 @@ function openTorrentContextMenu (infoHash) {
 }
 
 function getTorrentPath (torrentSummary) {
-  var itemPath = path.join(torrentSummary.path, torrentSummary.files[0].path)
-  if (torrentSummary.files.length > 1) {
-    itemPath = path.dirname(itemPath)
-  }
-  return itemPath
+  return path.join(torrentSummary.path, torrentSummary.name)
```

Comment:
I'll move that commit to its own PR.

--

Currently the Remove Data Files is pretty broken. If you drag, say, two files from your Downloads folder onto WebTorrent Desktop, create a torrent containing just those two, and then click Remove Data Files it'll remove your entire downloads folder (!)

I think the fix is to delete each file in the torrent individually